pentaho-js/src/main/javascript/cdf-legacy/cdf-base.js

```diff
@@ -193,3 +195,51 @@ wd.cdf.endpoints = {
     return enc;
   }
 };
+pho = typeof pho !== "undefined" ? pho : {};
```

Comment:
Please add an empty line before and after the `pho = ...` statement.

Reply:
done with requested changes